crates/nvim-api/src/ffi/buffer.rs

```diff
@@ -1,19 +1,13 @@
+#[cfg(feature = "neovim-nightly")]
+use luajit_bindings as lua;
```

Comment:
nit: not a fan of `cfg`-specific use statements, can you use the more verbose `*mut luajit_bindings::ffi::lua_State` instead?